README.md

```diff
@@ -110,6 +112,14 @@ target as shown in the figure below.
 </figure>
 </center>
 
+####b. Using CocoaPods
```

Comment:
Let's make a separate markdown file for dependency managers. Let's have a section for CocoaPods and a section for Carthage (see other commit). Here in the readme, let's just have a sentence like "You can also import ResearchKit into your project using a [dependency manager](link) such as CocoaPods or Carthage."

Then we can drop having the fourth-level headers.